<?xml version="1.0" encoding="utf-8"?>
<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
  <file datatype="xml" original="UpdateSourceExceptionFilterCallback.xml" source-language="en-US" target-language="ru-RU">
    <header>
      <tool tool-id="mdxliff" tool-name="mdxliff" tool-version="1.0-15c36f0" tool-company="Microsoft" />
      <xliffext:skl_file_name xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">02cd5861-7ce2-4a82-b358-31f8435a0ac565b5bfbadcdb56e0bc2f5373d8d94483568f6634.skl</xliffext:skl_file_name>
      <xliffext:version xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">1.2</xliffext:version>
      <xliffext:ms.openlocfilehash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">65b5bfbadcdb56e0bc2f5373d8d94483568f6634</xliffext:ms.openlocfilehash>
      <xliffext:ms.sourcegitcommit xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">d31dc2ede16f6f7bc64e90d9f897ff54c4e3869b</xliffext:ms.sourcegitcommit>
      <xliffext:ms.lasthandoff xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">04/03/2018</xliffext:ms.lasthandoff>
      <xliffext:moniker_ids xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">netframework-4.5.1,netframework-4.5.2,netframework-4.5,netframework-4.6.1,netframework-4.6.2,netframework-4.6,netframework-4.7.1,netframework-4.7</xliffext:moniker_ids>
    </header>
    <body>
      <group id="content" extype="content">
        <trans-unit id="101" translate="yes" xml:space="preserve" uid="T:System.Windows.Data.UpdateSourceExceptionFilterCallback">
          <source>The object with the exception.</source>
          
        </trans-unit>
        <trans-unit id="102" translate="yes" xml:space="preserve" uid="T:System.Windows.Data.UpdateSourceExceptionFilterCallback">
          <source>The exception encountered.</source>
          
        </trans-unit>
        <trans-unit id="103" translate="yes" xml:space="preserve" uid="T:System.Windows.Data.UpdateSourceExceptionFilterCallback">
          <source>Represents the method that handles exceptions that are thrown during the update of the binding source value.</source>
          
        </trans-unit>
        <trans-unit id="104" translate="yes" xml:space="preserve" uid="T:System.Windows.Data.UpdateSourceExceptionFilterCallback">
          <source>This must be used with the <ph id="ph1">&lt;see cref="T:System.Windows.Controls.ExceptionValidationRule" /&gt;</ph>.</source>
          
        </trans-unit>
        <trans-unit id="105" translate="yes" xml:space="preserve" uid="T:System.Windows.Data.UpdateSourceExceptionFilterCallback">
          <source>An object that is typically one of the following:</source>
          
        </trans-unit>
        <trans-unit id="106" translate="yes" xml:space="preserve" uid="T:System.Windows.Data.UpdateSourceExceptionFilterCallback">
          <source><ph id="ph1">&lt;list type="table"&gt;</ph><ph id="ph2">&lt;listheader&gt;</ph><ph id="ph3">&lt;term&gt;</ph> Value</source>
          
        </trans-unit>
        <trans-unit id="107" translate="yes" xml:space="preserve" uid="T:System.Windows.Data.UpdateSourceExceptionFilterCallback">
          <source><ph id="ph1">&lt;/term&gt;</ph><ph id="ph2">&lt;description&gt;</ph> Description</source>
          
        </trans-unit>
        <trans-unit id="108" translate="yes" xml:space="preserve" uid="T:System.Windows.Data.UpdateSourceExceptionFilterCallback">
          <source><ph id="ph1">&lt;/description&gt;</ph><ph id="ph2">&lt;/listheader&gt;</ph><ph id="ph3">&lt;item&gt;</ph><bpt id="p1">&lt;term&gt;</bpt><ph id="ph4">&lt;see langword="null" /&gt;</ph><ept id="p1">&lt;/term&gt;</ept><ph id="ph5">&lt;description&gt;</ph> To ignore any exceptions.</source>
          
        </trans-unit>
        <trans-unit id="109" translate="yes" xml:space="preserve" uid="T:System.Windows.Data.UpdateSourceExceptionFilterCallback">
          <source>The default behavior (if there is no <ph id="ph1">&lt;see cref="T:System.Windows.Data.UpdateSourceExceptionFilterCallback" /&gt;</ph>) is to create a <ph id="ph2">&lt;see cref="T:System.Windows.Controls.ValidationError" /&gt;</ph> with the exception and adds it to the <ph id="ph3">&lt;see cref="P:System.Windows.Controls.Validation.Errors" /&gt;</ph> collection of the bound element.</source>
          
        </trans-unit>
        <trans-unit id="110" translate="yes" xml:space="preserve" uid="T:System.Windows.Data.UpdateSourceExceptionFilterCallback">
          <source><ph id="ph1">&lt;/description&gt;</ph><ph id="ph2">&lt;/item&gt;</ph><ph id="ph3">&lt;item&gt;</ph><ph id="ph4">&lt;term&gt;</ph> Any object</source>
          
        </trans-unit>
        <trans-unit id="111" translate="yes" xml:space="preserve" uid="T:System.Windows.Data.UpdateSourceExceptionFilterCallback">
          <source><ph id="ph1">&lt;/term&gt;</ph><ph id="ph2">&lt;description&gt;</ph> To create a <ph id="ph3">&lt;see cref="T:System.Windows.Controls.ValidationError" /&gt;</ph> object with the <ph id="ph4">&lt;see cref="P:System.Windows.Controls.ValidationError.ErrorContent" /&gt;</ph> set to that object.</source>
          
        </trans-unit>
        <trans-unit id="112" translate="yes" xml:space="preserve" uid="T:System.Windows.Data.UpdateSourceExceptionFilterCallback">
          <source>The <ph id="ph1">&lt;see cref="T:System.Windows.Controls.ValidationError" /&gt;</ph> object is added to <ph id="ph2">&lt;see cref="P:System.Windows.Controls.Validation.Errors" /&gt;</ph> collection of the bound element.</source>
          
        </trans-unit>
        <trans-unit id="113" translate="yes" xml:space="preserve" uid="T:System.Windows.Data.UpdateSourceExceptionFilterCallback">
          <source><ph id="ph1">&lt;/description&gt;</ph><ph id="ph2">&lt;/item&gt;</ph><ph id="ph3">&lt;item&gt;</ph><ph id="ph4">&lt;term&gt;</ph> A <ph id="ph5">&lt;see cref="T:System.Windows.Controls.ValidationError" /&gt;</ph> object</source>
          
        </trans-unit>
        <trans-unit id="114" translate="yes" xml:space="preserve" uid="T:System.Windows.Data.UpdateSourceExceptionFilterCallback">
          <source><ph id="ph1">&lt;/term&gt;</ph><ph id="ph2">&lt;description&gt;</ph> To set the <ph id="ph3">&lt;see cref="T:System.Windows.Data.BindingExpression" /&gt;</ph> or <ph id="ph4">&lt;see cref="T:System.Windows.Data.MultiBindingExpression" /&gt;</ph> object as the <ph id="ph5">&lt;see cref="P:System.Windows.Controls.ValidationError.BindingInError" /&gt;</ph>.</source>
          
        </trans-unit>
        <trans-unit id="115" translate="yes" xml:space="preserve" uid="T:System.Windows.Data.UpdateSourceExceptionFilterCallback">
          <source>The <ph id="ph1">&lt;see cref="T:System.Windows.Controls.ValidationError" /&gt;</ph> object is added to <ph id="ph2">&lt;see cref="P:System.Windows.Controls.Validation.Errors" /&gt;</ph> collection of the bound element.</source>
          
        </trans-unit>
        <trans-unit id="116" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Windows.Data.UpdateSourceExceptionFilterCallback">
          <source>If you have associated the <ph id="ph1">&lt;xref:System.Windows.Controls.ExceptionValidationRule&gt;</ph> with your <ph id="ph2">&lt;xref:System.Windows.Data.Binding&gt;</ph> object you have the option to use the <ph id="ph3">&lt;xref:System.Windows.Data.Binding.UpdateSourceExceptionFilter%2A&gt;</ph> property to set this callback to provide custom logic for handling the exceptions.</source>
          <target state="translated">Если была связана <ph id="ph1">&lt;xref:System.Windows.Controls.ExceptionValidationRule&gt;</ph> с вашей <ph id="ph2">&lt;xref:System.Windows.Data.Binding&gt;</ph> объекта, у вас есть возможность использовать <ph id="ph3">&lt;xref:System.Windows.Data.Binding.UpdateSourceExceptionFilter%2A&gt;</ph> свойство, чтобы задать этот обратный вызов для предоставления пользовательской логики для обработки исключений.</target>       </trans-unit>
        <trans-unit id="117" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Windows.Data.UpdateSourceExceptionFilterCallback">
          <source>This callback is invoked whenever any exception is encountered when the binding engine updates the binding source value.</source>
          <target state="translated">Этот обратный вызов при обнаружении любое исключение, что механизму привязки при обновлении значения источника привязки.</target>       </trans-unit>
        <trans-unit id="118" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Windows.Data.UpdateSourceExceptionFilterCallback">
          <source>If an <ph id="ph1">&lt;xref:System.Windows.Data.Binding.UpdateSourceExceptionFilter%2A&gt;</ph> is not specified on the <ph id="ph2">&lt;xref:System.Windows.Data.Binding&gt;</ph>, the binding engine creates a <ph id="ph3">&lt;xref:System.Windows.Controls.ValidationError&gt;</ph> with the exception and adds it to the <ph id="ph4">&lt;xref:System.Windows.Controls.Validation.Errors%2A?displayProperty=nameWithType&gt;</ph> collection of the bound element.</source>
          <target state="translated">Если <ph id="ph1">&lt;xref:System.Windows.Data.Binding.UpdateSourceExceptionFilter%2A&gt;</ph> не указан в <ph id="ph2">&lt;xref:System.Windows.Data.Binding&gt;</ph>, обработчик привязки создает <ph id="ph3">&lt;xref:System.Windows.Controls.ValidationError&gt;</ph> с исключением и добавляет его в <ph id="ph4">&lt;xref:System.Windows.Controls.Validation.Errors%2A?displayProperty=nameWithType&gt;</ph> коллекции связанного элемента.</target>       </trans-unit>
        <trans-unit id="119" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Windows.Data.UpdateSourceExceptionFilterCallback">
          <source>The <ph id="ph1">&lt;xref:System.Windows.Controls.TextBox.Text%2A&gt;</ph> property of the following <ph id="ph2">&lt;xref:System.Windows.Controls.TextBox&gt;</ph> is data-bound to a source property <ph id="ph3">`Age3`</ph> that is of type <ph id="ph4">`int`</ph>.</source>
          <target state="translated"><ph id="ph1">&lt;xref:System.Windows.Controls.TextBox.Text%2A&gt;</ph> Свойство из следующих <ph id="ph2">&lt;xref:System.Windows.Controls.TextBox&gt;</ph> данных привязан к свойству источника <ph id="ph3">`Age3`</ph> типа <ph id="ph4">`int`</ph>.</target>       </trans-unit>
        <trans-unit id="120" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Windows.Data.UpdateSourceExceptionFilterCallback">
          <source>The <ph id="ph1">&lt;xref:System.Windows.Controls.ExceptionValidationRule&gt;</ph> checks for exceptions that are thrown during the update of the source property (such as when the user enters a value that cannot be converted to an integer).</source>
          <target state="translated"><ph id="ph1">&lt;xref:System.Windows.Controls.ExceptionValidationRule&gt;</ph> Проверяет наличие исключений, возникших во время обновления свойства источника (например, когда пользователь вводит значение, которое невозможно преобразовать в целое число).</target>       </trans-unit>
        <trans-unit id="121" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Windows.Data.UpdateSourceExceptionFilterCallback">
          <source>You have the option to provide custom logic to handle those exceptions.</source>
          <target state="translated">Имеется возможность для предоставления пользовательской логики для обработки этих исключений.</target>       </trans-unit>
        <trans-unit id="122" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Windows.Data.UpdateSourceExceptionFilterCallback">
          <source>The following example shows how to use the <ph id="ph1">&lt;xref:System.Windows.Data.Binding.UpdateSourceExceptionFilter%2A&gt;</ph> property to set an <ph id="ph2">&lt;xref:System.Windows.Data.UpdateSourceExceptionFilterCallback&gt;</ph>:</source>
          <target state="translated">В следующем примере показано, как использовать <ph id="ph1">&lt;xref:System.Windows.Data.Binding.UpdateSourceExceptionFilter%2A&gt;</ph> задаваемого свойства <ph id="ph2">&lt;xref:System.Windows.Data.UpdateSourceExceptionFilterCallback&gt;</ph>:</target>       </trans-unit>
        <trans-unit id="123" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Windows.Data.UpdateSourceExceptionFilterCallback">
          <source>The following is an example implementation of an <ph id="ph1">&lt;xref:System.Windows.Data.UpdateSourceExceptionFilterCallback&gt;</ph>:</source>
          <target state="translated">Ниже приведен пример реализации <ph id="ph1">&lt;xref:System.Windows.Data.UpdateSourceExceptionFilterCallback&gt;</ph>:</target>       </trans-unit>
        <trans-unit id="124" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Windows.Data.UpdateSourceExceptionFilterCallback">
          <source>For the complete sample, see <bpt id="p1">[</bpt>Binding Validation Sample<ept id="p1">](http://go.microsoft.com/fwlink/?LinkID=159972)</ept>.</source>
          <target state="translated">Полный пример см. в разделе <bpt id="p1">[</bpt>пример проверки привязки<ept id="p1">](http://go.microsoft.com/fwlink/?LinkID=159972)</ept>.</target>       </trans-unit>
      </group>
    </body>
  </file>
</xliff>